id-policy.md

```diff
@@ -0,0 +1,231 @@
+    manner.
+
+6.  URI documentation must be open so that it can be replicated and
+    reused.
+
+
+
+Individual ontology projects may, at their discretion, choose to manage
+these responses, with the understanding that if service lapses the
+Foundry may substitute the generic software for handling them in order
+to maintain service.
+
+## Policy for OBO Library ontologies
+
+OBO Library ontologies are not constrained by this policy, however, we
+recommend that they follow it nonetheless, for three reasons. First, it
+provides a uniform experience and sets expectations for ontology
+clients. Second, by doing so library ontologies will be able to take
+advantage of shared infrastructure. Third, ontologies that eventually
+join the foundry would have to disrupt their ids if they had to change
+to follow this policy.
+
+**Allocating IDSPACEs**
+
+IDSPACEs within the OBO library are unique for a given project and are
+chosen not to conflict with prefix for xrefs. Although IDSPACEs are
+case-sensitive, there will never be more than one IDSPACEs that are the
+same when compared in a case-insensitive manner. Therefore,although
+"GO" and "go", "Go" and "gO" are different IDSPACEs, the IDSPACE "go",
+"Go" and "gO" will not be used as "GO" has already been allocated.
+
+
+
+A registry of allocated IDSPACEs will be maintained. Requests for an
+IDSPACE should be made by sending mail to
+obo-discuss@lists.sourceforge.net, cc obo-admin@fruitfly.org. A request
+should include information about the ontology, such as scope and
+maintainer and a confirmation that the ontology is open access.
+
+
+
+
+**Current ontology document:**
+
+
+
+The most current version of an ontology will be at the following URL,
+where "IDSPACE" is replaced with the IDSPACE of the given ontology in
+lower case.
+
+
+
+ * Current OWL: http://purl.obolibrary.org/obo/idspace.owl
+ * Current OBO: http://purl.obolibrary.org/obo/idspace.obo
+
+
+For example, the Ontology for Biomedical Investigations has the IDSPACE
+"OBI", so the current version of the OWL document would be at
+http://purl.obolibrary.org/obo/obi.owl
+
+Additionally other ontology products can be included underneath this. For examples, see "products" under [http://obofoundry.org/ontology/go](http://obofoundry.org/ontology/go)
+
+## Versions of ontologies
+
+
+Versions are named by a date in the following format: YYYY-MM-DD. For a
+given version of an ontology, the ontology should be accessible at the
+following URL, where *idspace*; is replaced by the IDSPACE in lower
+case
+
+OWL:
+
+http://purl.obolibrary.org/obo/*idspace*/YYYY-MM-DD/*idspace*.owl
+
+OBO:
+http://purl.obolibrary.org/obo/*idspace*/YYYY-MM-DD/*idspace*.obo
+
+
+For example, for the version of OBI released 2009-11-06, the OWL
+document is accessible
+http://purl.obolibrary.org/obo/obi/2009-11-06/obi.owl
+
+
+## Home page
+
+
+If the ontology has a home page on the Web, it is accessible at
```

Comment:
we recommend it is accessible at...? I don't think this happens in practice. Similarly for tracker browser etc. Or is that auto generated from the registry entry?

Reply:
They are not auto-generated, and in practice this is only done for 35 ontologies, so I second the change of wording to a recommendation

Reply:
In fact, why recommend it at all? It's one more URL that can fall out of sync (for humans we have the tracker link in the obo metadata). I don't really see the use case for a PURL for a tracker. But can maybe be tackled on 2nd pass